component/src/main/ballerina/net.fs/annotation.bal

```diff
@@ -0,0 +1,13 @@
+package ballerina.net.fs;
```

Comment:
Please add license header to all.

Reply:
I have added to the new file. This local DirectoryListener module will move to the main ballerina repo.